css/styles.css

```diff
@@ -0,0 +1,147 @@
+/* Style.css */
+* {
+  box-sizing: border-box;
+}
+
+.clearfix:after {
+  content: "";
+  display: table;
+  clear: both;
+}
+
+body {
+  font-family: Century Gothic, sans-serif;
+  font-size: 16px;
+}
+
+h1 {
+  font-size: 3rem;
+  padding: 20px;
+  width: 100%;
+  text-align: center;
+
+}
+
+h1 a {
+  background-image: url("http://www.pexels.com/photo/food-raspberries-fruits-7329/");
+  background-size: cover;
+}
+
+h2 {
+  font-size: 1.5rem;
+  padding: 20px;
+  text-align: center;
+  background-color: lightgrey;
+  width: 100%;
+}
+
+.topics li {
+  font-size: 1rem;
+  float: left;
+  width: 25%;
+  margin-top: 15px;
+  margin-bottom: 5px;
+  padding-left: 5px;
+
+}
+
+.aboutmuse {
```

Comment:
Use snake case when creating your class names. "-" between two words to make it easier to read. Though, you kept your naming conventions consistent. Your way isn't necessarily wrong, but it's an accepted practice to put a dash.